docs/application/dotnet/guides/opentk/work-with-skiasharp.md

```diff
@@ -0,0 +1,183 @@
+# Work with SkiaSharp
+
+SkiaSharp is a cross-platform 2D graphics API for .NET platforms based on Google's Skia Graphics Library. 
```

Comment:
Please change to:

SkiaSharp is a cross-platform 2D graphics API for the .NET platforms based on the Skia Graphics Library of the Google.